lib/kafo/configuration.rb

```diff
@@ -210,7 +210,7 @@ class { '::kafo_configure::dump_values':
 
           if (version_mismatch = /kafo_configure::puppet_version_failure: (.+?\))/.match(stderr))
             puts version_mismatch[1]
-            puts "Cannot continue due to incompatible version of Puppet. Use --skip-puppet-version-check to disable this check."
+            puts 'Cannot continue due to incompatible version of Puppet. Use --skip-puppet-version-check to disable this check.'
```

Comment:
Did this change fix an issue?

Reply:
This got me thinking about something I read the other week. From a newer project around standardizing Ruby style. I think there used to be a performance issue, perhaps way back, with single and double quoted strings. Reading on this more and looking at style guides, it seems the consensus these days is there is no performance issue and just use double-quoted strings everywhere:

 * https://github.com/testdouble/standard#standardrb-the-rules
 * https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#strings
 * https://anti-pattern.com/always-use-double-quoted-strings-in-ruby

A few resources I was reading. I do realize style takes us down a whole nother Rabbit hole. So I'd encourage not changing unrelated strings style wise to prevent that. I have an item to discuss further.

Reply:
No, I just noticed that we don't need double quotes here since we aren't doing any interpolation. It didn't feel like it deserved a separate PR.

With that said, I did notice there are some rubocops we can fix, so I can drop this for now for clarity and address it with that change later.

Reply:
Thanks, I guess I had outdated information about single vs. double quotes. Based on this information, I'll update this to use double quotes throughout this PR.